.github/workflows/sphinx-build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install Dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install --editable .
+        SETUPTOOLS_ENABLE_FEATURES="legacy-editable" pip install --editable .
```

Comment:
`pip` might not be `python -m pip` -- there's no virtual environment here, so best to be specific any use `python -m pip`

Reply:
well, it did work, but change for consistency

Reply:
ok, well, I see it was reviewed already anyways